hash_practice/exercises.py

```diff
@@ -5,17 +5,56 @@ def grouped_anagrams(strings):
         Time Complexity: ?
         Space Complexity: ?
     """
-    pass
+    if len(strings) == 0:
+        return []
+    if len(strings) == 1:
+        return [[strings[0]]]
+    list1 = [strings[0]]
+    listOfLists = [list1]
+    for word in strings[1:]:
+        flag = False

```

Comment:
Likewise, this could be better named something like `matching_list_found` rather than `flag`.